src/components/HotspotsCarousel.tsx

```diff
@@ -33,26 +36,29 @@ const HotspotItem = ({ hotspot, totalReward }: HotspotsItemProps) => {
       borderRadius="m"
       marginStart="l"
       height={75}
+      onPress={() => navigation.navigate('HotspotDetails', { hotspot })}
     >
-      <Box flexDirection="column">
-        <Box flexDirection="row">
-          <CheckCircle width={17} height={17} />
-          <Text
-            variant="body2Medium"
-            color="black"
-            paddingStart="s"
-            width="80%"
-          >
-            {animalName(hotspot.address)}
+      <Box flexDirection="row">
+        <Box flexDirection="column">
+          <Box flexDirection="row">
+            <CheckCircle width={17} height={17} />
+            <Text
+              variant="body2Medium"
+              color="black"
+              paddingStart="s"
+              width="80%"
+            >
+              {animalName(hotspot.address)}
+            </Text>
+          </Box>
+          <Text variant="body2" color="purpleMain" paddingTop="s">
+            {`+${totalReward.toFixed(2)} HNT`}
```

Comment:
is this pulling from real data yet? Whenever it's wired up, we'll want to make sure to use helium-js for any calculations and `toString`

Reply:
It is, these arent coming as a balance from Helium JS though. I can make that update. The numbers we get are all in different formats, so I left them raw. I guess it probably makes more sense to convert them all to a Balance?

For example:
```
"data": {
"total": 10.98530262,
"sum": 1098530262,
"stddev": 0.922213400554,
"min": 0,
"median": 2.027979885,
"max": 2,
"avg": 1.83088377
}
```

Reply:
Yeah we should be doing any price math and string formatting with the `Balance` object because it can handle big numbers and formatting/rounding as a string

Reply:
Ok let me make that change